internal/blocksync/pool_test.go

```diff
@@ -43,7 +43,13 @@ func (p testPeer) runInputRoutine() {
 // Request desired, pretend like we got the block immediately.
 func (p testPeer) simulateInput(input inputData) {
 	block := &types.Block{Header: types.Header{Height: input.request.Height}}
-	input.pool.AddBlock(input.request.PeerID, block, 123)
+	blockID := types.BlockID{
+		Hash:          []byte{},
+		PartSetHeader: types.PartSetHeader{},
+	}
+	extCommitSigs := make([]types.ExtendedCommitSig, 0)
```

Comment:
Generally unless we know the exact eventual capacity, better to just initialize the zero value:

```suggestion
	var extCommitSigs []types.ExtendedCommitSig
```